sdk/attestation/azure-security-attestation/azure/security/attestation/_administration_client.py

```diff
@@ -4,316 +4,736 @@
+            for an isolated attestation instance.
+
+            .. literalinclude:: ../samples/sample_get_set_policy.py
+                :start-after: [BEGIN get_policy_management_certificate]
+                :end-before: [END get_policy_management_certificate]
+                :language: python
+                :dedent: 8
+                :caption: Retrieving the policy management certificates.
+
         """
 
+        # Merge our existing config options with the options for this API call.
+        # Note that this must be done before calling into the implementation
+        # layer because the implementation layer doesn't like keyword args that
+        # it doesn't expect :(.
+        options = merge_validation_args(
+            self._config._kwargs, kwargs  # pylint: disable = protected-access
+        )
+
         cert_response = self._client.policy_certificates.get(**kwargs)
-        token = AttestationToken[PolicyCertificatesResult](
-            token=cert_response.token,
-            body_type=PolicyCertificatesResult)
-        if self._config.token_validation_options.validate_token:
-            if not token.validate_token(self._config.token_validation_options, self._get_signers(**kwargs)):
-                raise Exception("Token Validation of PolicyCertificates API failed.")
+        token = AttestationToken(
+            token=cert_response.token, body_type=GeneratedPolicyCertificatesResult
+        )
+
+        if options.get("validate_token", True):
+            token._validate_token(  # pylint: disable = protected-access
+                self._get_signers(**kwargs), **options
+            )
         certificates = []
 
-        cert_list = token.get_body()
+        cert_list = token._get_body()  # pylint: disable = protected-access
 
         for key in cert_list.policy_certificates.keys:
-            key_certs = [base64.b64decode(cert) for cert in key.x5_c]
+            key_certs = [pem_from_base64(cert, "CERTIFICATE") for cert in key.x5_c]
             certificates.append(key_certs)
-        return AttestationResponse(token, certificates)
+        return certificates, token
 
     @distributed_trace
-    def add_policy_management_certificate(self, certificate_to_add, signing_key, **kwargs):
-        #type:(bytes, AttestationSigningKey, **Any) -> AttestationResponse[PolicyCertificatesModificationResult]
-        """ Adds a new policy management certificate to the set of policy management certificates for the instance.
+    def add_policy_management_certificate(self, *args, **kwargs):
+        # type: (*str, **Any) -> Tuple[AttestationPolicyCertificateResult, AttestationToken]
+        # pylint: disable = line-too-long
+        """Adds a new policy management certificate to the set of policy management certificates for the instance.
 
-        :param bytes certificate_to_add: DER encoded X.509 certificate to add to 
+        :param str certificate_to_add: PEM encoded X.509 certificate to add to
```

Comment:
Since the positional parameter is presented in a non-standard way through `*args`, it's probably good to emphasize that it's a required parameter
```suggestion
        :param str certificate_to_add: Required. PEM encoded X.509 certificate to add to
```